docs/maps/ravine.md

```diff
@@ -0,0 +1,40 @@
+<!-- replace _map_ with the actual map name -->
+<!-- change gamemode type for the Map data description  -->
+# Ravine
+
+***
+
+#### ![ravineicon](../assets/maps/ravine/ravine-icon.jpg)
+
+# Overview
+***
+- **Introduced:** v1.7.0
+- **Description:** A chasm that seperates two bases apart.
```

Comment:
Could this be renamed to something like: "A battlefield with a large chasm dividing the land."?